src/Gui/DlgSettingsLazyLoadedImp.h

```diff
@@ -58,6 +58,10 @@ protected Q_SLOTS:
 private:
     std::unique_ptr<Ui_DlgSettingsLazyLoaded> ui;
     static const uint WorkbenchNameRole;
+
+    std::vector<std::string> _backgroundAutoloadedModules;
+    std::string _startupModule;
+    std::map<QString, QCheckBox*> _autoloadCheckboxes;
```

Comment:
You seem to have forgotten to include QCheckBox in this header file.

Reply:
I just pushed a fix with a forward declaration of QCheckBox.